src/frontend/components/administration/apikey/ApikeyTab.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <v-card>
+    <v-toolbar color="secondary" dark flat dense>
+      <v-toolbar-title v-text="$t('administration.apikey.title')" />
+      <v-spacer />
+      <v-icon>mdi-security</v-icon>
+    </v-toolbar>
+    <v-card-actions>
+      <v-spacer />
+      <v-btn
+        @click.stop="setVisible(true)"
+        v-text="$t('administration.apikey.buttonCreate')"
+      />
+
+      <CreateDialog
+        :visible="createDialogVisible"
+        @created="getApikeys()"
+        @closed="setVisible(false)"
+      />
+    </v-card-actions>
+
+    <v-row
+      v-if="apikeys.length === 0"
+      align="center"
+      justify="center"
+      class="ma-2"
+    >
+      <v-col class="text-center" cols="12" sm="4">
+        {{ $t("administration.apikey.noApikeys") }}
+      </v-col>
+    </v-row>
+    <v-row v-else class="ma-2">
+      <v-col
+        v-for="(apikey, id) in apikeys"
+        :key="id"
+        cols="12"
+        sm="12"
+        md="12"
+        lg="6"
+        xl="6"
+      >
+        <ApikeyCard
+          :apikey="Object.keys(apikey)[0]"
+          :config="apikey[Object.keys(apikey)[0]]"
```

Comment:
If you need something that complicated to separate the ID and the body of the key, then you should rethink the structure of the server response instead of struggling with it on client side. If the server returns an array of objects, then include an `id` property instead of using the value as a root property.

Reply:
On another note, isn't the same thing as doing `Object.values(apikey)[0]` ?

Reply:
PR: https://github.com/ezpaarse-project/ezunpaywall/pull/112